site/content/docs/mdx.md

```diff
@@ -0,0 +1,288 @@
+</Callout>
+
+### Markdown
+
+As we've mentioned, MDX supports CommonMark syntax by default. On top of that Flowershow provides support for GFM and some Obsidian-specific extensions. Check our [syntax|Syntax guide] to learn which other Markdown syntax elements are supported by Flowershow.
+
+### JSX and custom components
+
+JSX allows you to create components, that you can reuse across all of your markdown files. This is especially useful if you find yourself writing a lot of custom HTML in your Markdown, and/or you use the same HTML code parts and copy them again and again to different markdown pages. You soon may find it difficult to tell what a given part of HTML block is supposed to do. And what used to be e.g. your blog content ends up as a Markdown-HTML-CSS soup. On top of that, imagine if you wanted to make some changes to all these copies of HTML blocks, that you used many different pages 🤯.
+
+Fortunately, thanks to JSX you can extract common HTML parts and enclose them in components, which you can then import and use wherever you want in your Markdown. What you end up with is much cleaner content and a single source of truth for HTML (JSX) parts you used to copy over and over again. JSX components allow you to declutter your Markdown and to separate content writing from its structuring and styling 😌. 
+
+For example, by creating a component out of a part of HTML code you repeat all over again, you could turn this...
+
+```md
+# Some page heading
+
+Some content here...
+
+<a href='https://some.url.com' className="no-underline">
+  <button
+    type="button"
+    class="inline-flex items-center px-2.5 py-1.5 border border-transparent text-xs font-medium rounded shadow-sm text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
+  >
+    Click me!
+  </button>
+</a>
+
+Some more content here...
+```
+
+...into to this:
+
+
+```md
+# Some page heading
+
+Some content here...
+
+<LinkButton href="https://some.url.com" text="Click me!" />
+
+Some more content here...
+```
+
+Looks much better! And it's much easier to update if you're using this button on many different pages.
+
+(We'll learn more about how to create, import, and use your custom components in the sections below.)
+
+### ESM `import` and `export` statements
+
+This is where a lot of MDX power comes from: support for ESM (ECMAScript Modules) `import` and `export` statements, which you can use to import custom JSX (React) components or even define them locally, as well as import and locally define data (variables), that can then be passed to your JSX components.
+
+#### Importing components
+
+To see this in action, let's create a React component in the `/components` folder:
```

Comment:
🚩 where is the `components` folders? This is crucial info for someone following along (is it relative to their `content` directory or where?

Atm it looks like this components folder is inside templates/default. Is there any way to import component relative to content folder or this file and have components in `site/content/components` or similar? That would be much more realistic for end users who don't have access to templates/default.

Reply:
I agree with you and I've tried it before. I'll double-check it but I think there is an error related to having mixed content in the /content folder.